autoload/EasyMotion.vim

```diff
@@ -399,7 +405,11 @@ function! s:Prompt(message) " {{{
     echohl None
 endfunction " }}}
 function! s:Throw(message) "{{{
-    throw 'EasyMotion: ' . a:message
+    if g:EasyMotion_verbose
```

Comment:
Even if `&verbose == 0`, Vim throws exceptions, so I don't want to suppress the error message.
It makes it hard to debug.